packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -341,7 +342,8 @@ export class DashboardService {
 
         const duplicatedDashboard = {
             ...dashboard,
-            name: `Copy of ${dashboard.name}`,
+            description: data.dashboardDesc,
+            name: data.dashboardName,
```

Comment:
actually, because of this change I think I'm going to add duplicate modal to dashboard and chart pages too..

Reply:
I just realized that places that call the duplicate mutation directly - won't have `copy - ` .. in their title